### PR TITLE
Pr doc tn

### PR DESCRIPTION
**Pull request by yzhang123**
# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 Text (Inverse) Normalization
 ============================
 
+.. warning::
+
+    *TN/ITN is transitioning from [NVIDIA/NeMo](https://github.com/NVIDIA/NeMo) repository to a standalone [NVIDIA/NeMo-text-processing](https://github.com/NVIDIA/NeMo-text-processing) repository. All updates and discussions/issues should go to the new repository.*
```
Comment (ekmb) [inline docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst:8]: `is transitioning` -> `transitioned`

File: docs/source/nlp/text_normalization/wfst/wfst_text_processing_deployment.rst [hunk 2]
```diff
@@ -3,9 +3,14 @@
 Deploy to Production with C++ backend
 =====================================
 
-NeMo provides tools to deploy :doc:`TN and ITN <wfst_text_normalization>` for production :cite:`textprocessing-deployment-zhang2021nemo`.
+.. warning::
+
+    *TN/ITN is transitioning from [NVIDIA/NeMo](https://github.com/NVIDIA/NeMo) repository to a standalone [NVIDIA/NeMo-text-processing](https://github.com/NVIDIA/NeMo-text-processing) repository. All updates and discussions/issues should go to the new repository.*
```
Comment (ekmb) [inline docs/source/nlp/text_normalization/wfst/wfst_text_processing_deployment.rst:8]: same

**Review by ekmb — CHANGES_REQUESTED** [review]

File: docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst (hunk 1 above)
Comment (yzhang123) [reply docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst]: done

File: docs/source/nlp/text_normalization/wfst/wfst_text_processing_deployment.rst (hunk 2 above)
Comment (yzhang123) [reply docs/source/nlp/text_normalization/wfst/wfst_text_processing_deployment.rst]: updated

**Comment by yzhang123** [comment]
the DCO was not me, so i can't fix it

**Review by ekmb — APPROVED** [review]

File: docs/source/nlp/text_normalization/wfst/wfst_customization.rst [hunk 3]
```diff
@@ -3,9 +3,9 @@
 Grammar customization
 =====================
 
-.. note::
+.. warning::
 
-    TN/ITN is transitioning from `NVIDIA/NeMo <https://github.com/NVIDIA/NeMo>`_ repository to a standalone `NVIDIA/NeMo-text-processing <https://github.com/NVIDIA/NeMo-text-processing>`_ repository. All updates and discussions/issues should go to the new repository.
+    TN/ITN transitioned from `NVIDIA/NeMo <https://github.com/NVIDIA/NeMo>`_ repository to a standalone `NVIDIA/NeMo-text-processing <https://github.com/NVIDIA/NeMo-text-processing>`_ repository. All updates and discussions/issues should go to the new repository.
```
Comment (XuesongYang) [inline docs/source/nlp/text_normalization/wfst/wfst_customization.rst:8]: nit: are we planning to remove these sections in the NeMo doc, or cross-ref the NeMo text processing repo doc here? if it is the former, please specify which release we will remove it. For example, we could say "we will remove this section on the release 1.17.0. All updates..."

File: docs/source/nlp/text_normalization/wfst/wfst_customization.rst [hunk 4]
```diff
@@ -15,8 +15,8 @@ Steps to customize grammars
...
-3. Modify existing grammars or add new grammars to cover the target case using `Tutorial on how to write new grammars <https://colab.research.google.com/github/NVIDIA/NeMo/blob/stable/tutorials/text_processing/WFST_Tutorial.ipynb>`_
+2. Run `nemo_text_processing/text_normalization/normalize.py <https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/text_normalization/normalize.py>`_ or `nemo_text_processing/inverse_text_normalization/inverse_normalize.py <https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/inverse_text_normalization/inverse_normalize.py>`_ with `--verbose` flag to evaluate current behavior on the target case, see argument details in the scripts and `this tutorial <https://colab.research.google.com/github/NVIDIA/NeMo-text-processing/blob/main/tutorials/Text_(Inverse)_Normalization.ipynb>`_
```
Comment (XuesongYang) [inline docs/source/nlp/text_normalization/wfst/wfst_customization.rst:18]: nit: do we want to point at `main` or `stable`?

**Review by XuesongYang — APPROVED** [review]
Thank you! left some nit picks.

File: docs/source/nlp/text_normalization/wfst/wfst_customization.rst (hunk 3 above)
Comment (yzhang123) [reply docs/source/nlp/text_normalization/wfst/wfst_customization.rst:8]: the latter

File: docs/source/nlp/text_normalization/wfst/wfst_customization.rst (hunk 4 above)
Comment (yzhang123) [reply docs/source/nlp/text_normalization/wfst/wfst_customization.rst:18]: main, there is no stable in new repo yet

**Review by ekmb — APPROVED** [review]

**Comment by XuesongYang** [comment]
> the DCO was not me, so i can't fix it

overrided and merged.

**Pull request merged**